### PR TITLE
Add an event and presentation

### PR DESCRIPTION
**Pull request by jw1540**

### CODE REVIEW

File: Resources/Views/Authentication/create_events.leaf [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#extend("base"):
+    #export("title", "SwiftLeeds")
+    #export("content"):
+        #extend("Shared/_header")
+        <div class="row">
+            <form method="POST" action="/api/v1/events/create" enctype="application/x-www-form-urlencoded">
+                <h1>Create presentation</h1>
```
Comment (pdamonkey) [inline Resources/Views/Authentication/create_events.leaf:7]: Should be Create event

File: Resources/Views/Authentication/create_events.leaf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                <br>
+                <div class="mb-3">
+                  <label for="title" class="form-label">Event Name</label>
```
Comment (pdamonkey) [inline Resources/Views/Authentication/create_events.leaf:10]: Do we need Event in front of these as we know we're on a Create Event page, i.e. just Name?  Same for other labels

File: Resources/Views/Authentication/create_events.leaf [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                  <label for="location" class="form-label">Event Location</label>
+                  <input class="form-control" name="location" id="location">
+                </div>
+                <button type="submit" class="btn btn-primary">Submit</button>
+            </form>
+            <div class="col-6">
+              <img src="/img/header-bg.png" class="child-img">
+            </div>
+        </div>
+        <script src="https://cdn.jsdelivr.net/npm/@easepick/bundle@1.2.0/dist/index.umd.min.js"></script>
```
Comment (pdamonkey) [inline Resources/Views/Authentication/create_events.leaf:28]: This is already in base

File: Resources/Views/base.leaf [hunk 4]
```diff
@@ -21,6 +21,8 @@
     <script src='https://js.tito.io/v2' async></script>
     
     <script defer data-domain="swiftleeds.co.uk" src="https://plausible.io/js/plausible.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@easepick/bundle@1.2.0/dist/index.umd.min.js" type="text/javascript" async=""></script>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
```
Comment (pdamonkey) [inline Resources/Views/base.leaf:25]: What is jquery used for?

File: Resources/Views/Authentication/create_events.leaf (hunk 3 above)
Comment (pdamonkey) [inline Resources/Views/Authentication/create_events.leaf:28]: #import("scripts") in the base to their so you can extend it to separate out the javascript from the html
